micronaut-camunda-bpm-feature/transactional-test-data-jdbc/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id "io.micronaut.library"
```

Comment:
It's great that you found out that it's the kts file which triggered the bug.

In my opinion you have now undone more than necessary.
* Basically it is enough to rename the file and make a few small adjustments, see https://github.com/tobiasschaefer/micronaut-camunda-bpm/commit/8c0ecc1b53f4af5da5ca1337a5f99aadfbf62861
* Please also add the following to the topmost gradle.properties, so that a parallel build is always executed:
`org.gradle.parallel=true` (I prefer this over the --parallel argument, because then it is the default and does not have to be specified explicitly), see also https://docs.gradle.org/current/userguide/build_environment.html



Translated with www.DeepL.com/Translator (free version)